### DISCUSSION TITLE
Pessoas de dev. para seguir

### DISCUSSION THREAD

**Discussion by felipeorlando** [frontendbr/forum#539] [Dicas]
Gostaria de sugestões de pessoas a se seguir na comunidade de desenvolvimento, tanto no GitHub quanto no Twitter. Principalmente para descobrir novidades direto da fonte. 👍 

Pessoas da comunidade BR que são ativos também são bem vindos. 😄

**Comment by wellingtonpgp** [comment]
https://www.facebook.com/diariobackend

**Comment by andersoonweb** [comment]
@felipeorlando criaram esse guia [aqui](https://uptodate.frontendrescue.org/). Já tem um bom tempo, mas acho válido ainda.

**Comment by gabrieldarezzo** [comment]
Eu peguei umas referencias legais do https://github.com/Mendrone/Feed-De-Leitura
